lib/rex/parser/arguments.rb

```diff
@@ -12,21 +12,42 @@ module Parser
     #
     ###
     class Arguments
+      # e.g. '-x' or '-xyz'
+      SHORT_FLAG = /^-[a-zA-Z]+$/.freeze
```

Comment:
I feel like this is a plural, but not strongly opinionated

```suggestion
      SHORT_FLAGS = /^-[a-zA-Z]+$/.freeze
```